.github/workflows/test-ci.yaml

```
@@ -0,0 +1,60 @@
+
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile
+
+      - name: Run lint
+        run: pnpm run lint
+
+      - name: Run tests
+        run: pnpm run test
+
+      - name: Run build
+        run: pnpm run build
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)